zuul.d/timer_pipelines.yaml

```diff
@@ -18,3 +18,12 @@
     trigger:
       timer:
         - time: '0 3 * * *'
+
+- pipeline:
+    name: periodic-midnight
+    description: This pipeline runs jobs daily at 0am.
+    post-review: true
+    manager: independent
+    trigger:
+      timer:
+        - time: '0 0 * * *'
```

Comment:
this may run twice if we alternate between summer and normal time, no?

Reply:
Should not be a big deal.

Reply:
all times are in UTC